UNIXFS.md

```diff
@@ -90,7 +103,9 @@ UnixFS currently supports two optional metadata fields:
   - The remaining 20 bits are reserved for future use, and are subject to change. Spec implementations **MUST** handle bits they do not expect as follows:
     - For future-proofing the (de)serialization layer must preserve the entire uint32 value during clone/copy operations, modifying only bit values that have a well defined meaning: `clonedValue = ( modifiedBits & 07777 ) | ( originalValue & 0xFFFFF000 )`
     - Implementations of this spec must proactively mask off bits without a defined meaning in the implemented version of the spec: `interpretedValue = originalValue & 07777`
-* `mtime` -- The modification time in seconds since the epoch. This defaults to the unix epoch if unspecified
+* `mtime` -- A two-element structure ( `EpochSeconds`, `EpochNanoseconds` ) representing the modification time in seconds relative to the unix epoch `1970-01-01T00:00:00Z`. In contexts where an mtime is mandatory ( e.g. FUSE interfaces ) implementations must treat an unspecified mtime as `0`.
```

Comment:
I think we should just default this to `0` as it is currently.  The `go-ipfs` and `js-ipfs` implementations cannot control which context they will be used from so should default to the most conservative approach so that might as well be in the spec.

Reply:
My hesitation placing this in the spec is that it implicitly means `unknown mtime is not a thing`. E.g. a web gateway would be obligated to send out a `0` `Last-Modified` header, and so on...

What do you think?

Reply:
I think that's fine.  At least, I can't think of a reason that makes it not fine, given that [we need a default value](https://github.com/ipfs/specs/pull/232#discussion_r360469535) and we're happy with seeing 1970 everywhere when we `ipfs files ls -l`.

Reply:
To make this more concrete, I think a web gateway sending a `0` `Last-Modified` header is ok.

Reply:
:(

I will relent if this is a wider consensus, but then it makes me sad as this part of the spec becomes unusable to me as well. In my use case having a gateway render "this is the mtime" vs "mtime was not supplied" is really crucial.